src/main/kotlin/org/dashj/platform/dapiclient/DapiClient.kt

```diff
@@ -547,147 +546,34 @@ class DapiClient(
      */
     fun getIdentitiesByPublicKeyHashes(pubKeyHashes: List<ByteArray>, prove: Boolean = false):
         GetIdentitiesByPublicKeyHashesResponse {
-            logger.info("getIdentitiesByPublicKeyHashes(${pubKeyHashes.map { it.toHex() }}, $prove")
-            val method = GetIdentitiesByPublicKeyHashes(pubKeyHashes, prove)
-            val response = grpcRequest(method) as PlatformOuterClass.GetIdentitiesByPublicKeyHashesResponse
-
-            return when {
-                prove && response.hasProof() -> {
-                    val proof = Proof(response.proof)
-                    logger.info("proof = $proof")
-                    val (inclusion, noninclusion) = verifyProof(proof, ResponseMetadata(response.metadata), method)
-
-                    if (inclusion.isNotEmpty()) {
-                        // determine the pubKeyHashes not found
-                        GetIdentitiesByPublicKeyHashesResponse(
-                            inclusion.values.toList(), proof,
-                            ResponseMetadata(response.metadata)
-                        )
-                    } else {
-                        GetIdentitiesByPublicKeyHashesResponse(listOf(), proof, ResponseMetadata(response.metadata))
-                    }
-                }
-                else -> {
-                    val inclusion = arrayListOf<Identity>()
-                    val notFoundPublicKeys = arrayListOf<ByteArray>()
-                    val identityList = response.identitiesList.map {
-                        val item = Cbor.decodeList(it.toByteArray())
-                        if (item.isNotEmpty()) {
-                            item[0] as ByteArray
-                        } else {
-                            ByteArray(0)
-                        }
-                    }
-                    GetIdentitiesByPublicKeyHashesResponse(
-                        identityList, Proof(response.proof),
-                        ResponseMetadata(response.metadata)
-                    )
-                }
-            }
-        }
+        logger.info("getIdentitiesByPublicKeyHashes(${pubKeyHashes.map { it.toHex() }}, $prove")
+        val method = GetIdentitiesByPublicKeyHashes(pubKeyHashes, prove)
+        val response = grpcRequest(method) as PlatformOuterClass.GetIdentitiesByPublicKeyHashesResponse
 
-    /**
-     * Fetch the identity id by the first public key hash
-     * @param pubKeyHash ByteArray
-     * @param prove Whether to return the proof
-     * @return String
-     */
-    fun getIdentityIdByFirstPublicKey(pubKeyHash: ByteArray, prove: Boolean = false): ByteArray? {
-        logger.info("getIdentityIdByFirstPublicKey(${pubKeyHash.toHex()}, $prove)")
-        val method = GetIdentityIdsByPublicKeyHashes(listOf(pubKeyHash), prove)
-        val response = grpcRequest(method) as PlatformOuterClass.GetIdentityIdsByPublicKeyHashesResponse?
```

Comment:
these diffs look strange, but really we were removing `getIdentityId*` methods because they have been removed in Platform 0.23 and code is similar in neighboring methods.